site/content/contribute/webapp/redux/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Redux"
+heading: "Redux"
+description: "mattermost-redux is a library that contains logic for loading and working with data from the Mattermost server."
+date: "2020-04-01T12:00:00-04:00"
+section: "contribute"
+weight: 8
+---
+
+[mattermost-redux](https://github.com/mattermost/mattermost-webapp/tree/master/packages/mattermost-redux) is a library that contains logic for loading and working with data from the Mattermost server. It's currently used primarily by the Mattermost web app and by some plugins, and it was previously used by the Mattermost mobile app.
+
+As part of our current effort to turn [the mattermost-webapp repository](https://github.com/mattermost/mattermost-webapp) into a monorepo, we are also re-evaluating the purpose of mattermost-redux and how we can better expose its functionality for use in third party integrations. The structure of the library and what it exposes for integrations may change over time because of this.
```

Comment:
should this have its own section or a warning icon? I feel people will skip it as if it was part of the "intro"

Reply:
I don't know if there's a good way to add a warning icon since this is plain markdown. I'm not sure of how to call it out better in a way that makes it sound like we have immediate plans here

Reply:
How about "**Note:**?

Reply:
Yeah, that's a simple enough fix. I'll add that